Wright_Will/Assignments/ParseCSV.py

```diff
@@ -0,0 +1,41 @@
+## parse csv
+
+import csv
+import pprint
+import re
+with open("us-500.csv","rU") as csvfile:
+    read = csv.reader(csvfile)
```

Comment:
Perfect.

I had one of my students in the previous cohort take on this challenge, so I figure you might be interested in trying it out too. Currently, you're using python's built-in csv module for parsing. How would you go about writing a .csv parser yourself?

At first, it seems deceptively easy; but keep in mind, you might have a field that has a " in it, or a ' in it, or perhaps even a , in it. How would you handle these situations?

=]